src/lib.rs

```diff
@@ -16,8 +16,8 @@
 #![allow(clippy::many_single_char_names)]
 
 pub use poly::{
-    EmitLines, Line, Lines, MapToVertices, MapVertex, Polygon, Quad, Triangle, Vertices,
-    VerticesIterator,
+    EmitLines, EmitVertices, Line, Lines, LinesIterator, MapToVertices, MapToVerticesIter,
+    MapVertex, Polygon, Quad, Triangle, Vertices, VerticesIterator,
```

Comment:
I also found out that the trait `EmitVertices` and the structs `LinesIterator` and `MapToVerticesIter` weren't publicly re-exported. I assume this was an oversight and wasn't meant to be the case? If not I can revert this change of course.